src/main/scala/uk/gov/nationalarchives/tdr/api/model/consignment/CreateConsignmentReference.scala

```diff
@@ -0,0 +1,16 @@
+package uk.gov.nationalarchives.tdr.api.model.consignment
+
+import uk.gov.nationalarchives.oci.{Alphabet, BaseCoder, IncludedAlphabet}
+
+class CreateConsignmentReference {
```

Comment:
Good point! I'll change it to an object, makes using it easier too since I then don't have to have it as a constructor to the `consignmentService` :raised_hands: